src/NuGet.Core/NuGet.Packaging/PackageCreation/Authoring/PackageBuilder.cs

```diff
@@ -177,6 +177,12 @@ public bool RequireLicenseAcceptance
             set;
         }
 
+        public bool SuppressRequireLicenseAcceptance
```

Comment:
`EmitRequireLicenseAcceptance` or others might be better.

Reply:
I think we should that too. 
The default is more natural there. 

cc @dominoFire do you agree?

Reply:
Let me know if you would like me to do this.

Reply:
@dominoFire I will go ahead with this unless you see a problem with it.

Reply:
This is done in the last commit.